build-tools/packages/build-cli/src/handlers/checkFunctions.ts

```diff
@@ -28,6 +30,12 @@ import { getRunPolicyCheckDefault } from "../repoConfig.js";
 import { FluidReleaseStateHandlerData } from "./fluidReleaseStateHandler.js";
 import { BaseStateHandler, StateHandlerFunction } from "./stateHandlers.js";
 
+/**
+ * Only client and server release groups use changesets and the related release note and per-package changelog
+ * generation. Other release groups use various other means to track changes.
+ */
+const releaseGroupsUsingChangesets = new Set(["client", "server"]);
```

Comment:
There aren't too many, no. Most have some sort of related config already or are centralized, like the "default bump types for branch" function. In the new build-infrastructure APIs, release groups have an obvious place in config where settings like this can live. That's possible today, but it's more formalized in the new tools. (In case you're interested: https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-infrastructure/README.md)

Long term I'd like to see the "smart" build-tools go away. I think flub release does way too much, and we'd be better off with smaller individual purposeful commands and well-documented process that uses them.

My position has definitely shifted since flub release was originally written. There are things it does that are helpful, like checks to ensure things aren't missed (e.g. the ones in this PR), but the encoding of the whole process into one command isn't great.